caffe2/python/models/download.py

```diff
@@ -80,8 +80,6 @@ def downloadFromURLToFile(url, filename, show_progress=True):
     except URLError as e:
         raise Exception("Could not download model. [URL Error] {reason}."
```

Comment:
Can we just use a custom exception class here instead of having to catch all exceptions in c2_ref_test?

Reply:
...or maybe just remove this exception wrapping entirely, since it seems like it just obfuscates the actual error without providing extra value?

Reply:
I agree with you, if it were something that needs to be maintained going forward, but hopefully this code will be deprecated in next 6-9 month. 
And if this code were python3 only, than using exception chaining (i.e. `raise Exception() from e`) perhaps would be the right thing to do.